server/src/main.go

```diff
@@ -240,6 +240,7 @@ func router() (*mux.Router, *sqlx.DB) {
 	router.Handle("/list_connections", alice.New(main.authOrganizerMiddleware).ThenFunc(main.ListConnectionsHandler))
 	router.Handle("/list_activists", alice.New(main.authOrganizerOrNonSFBayMiddleware).ThenFunc(main.ListActivistsHandler))
 	router.Handle("/new_activists", alice.New(main.authOrganizerOrNonSFBayMiddleware).ThenFunc(main.NewActivistsHandler))
+	router.Handle("/new_activists_pending_workshop", alice.New(main.authOrganizerMiddleware).ThenFunc(main.NewActivistsPendingWorkshopHandler))
```

Comment:
were you intending this to be for SF bay only or all chapters?

if all chapters, i think you either need authOrganizerOrNonSFBayMiddleware here and apiOrganizerOrNonSFBayAuthMiddleware below, otherwise need to add "roleRequired": ["organizer"] in shared/nav.json to make it appear only in the SF bay chapter (since "organizer" role currently effectively means "sf bay organizer")